packages/composite-checkout/src/components/checkout.js

```diff
@@ -120,7 +123,21 @@ export default function Checkout( { steps, className } ) {
 	} );
 	const isThereAnotherNumberedStep = !! nextStep && nextStep.hasStepNumber;
 	const isThereAnIncompleteStep = !! annotatedSteps.find( step => ! step.isComplete );
-	const isCheckoutInProgress = isThereAnIncompleteStep || isThereAnotherNumberedStep;
+	const isCheckoutInProgress =
+		isThereAnIncompleteStep || isThereAnotherNumberedStep || formStatus !== 'ready';
+
+	if ( formStatus === 'loading' ) {
+		return (
```

Comment:
This code doesn't seem to be running for me.

Reply:
Is it possible that you were using calypso instead of the demo? I hadn't added the loading state to the calypso code yet, but I really should have, so thanks for mentioning it. Does it work now?

Reply:
Yes I was. I should have read the instructions better. Sorry my bad :P 

It works now thank you!